### PR TITLE
[*] Query Builder - Fix broken queries on collection having integrations

### PR DESCRIPTION
**Pull request by larcin**
## Pull Request checklist:

- [ ] Write an explicit title for the Pull Request
- [ ] Write changes made in the CHANGELOG.md
- [ ] Test manually the implemented changes
- [ ] Review my own code (indentation, syntax, style, simplicity, readability)
- [ ] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by larcin** [comment]
To test:
- Setup a stripe integration on a mongo table: https://docs.forestadmin.com/documentation/reference-guide/integrations#integrations
- See that you can't access the so called table.
- Move on to my branch
- It works.

**Comment by SteveBunlon** [comment]
[P1 - Integartions - Users can't load "users" collection when an integration is set up](https://trello.com/c/fWMtYoMM/3764-p1-integartions-users-cant-load-users-collection-when-an-integration-is-set-up)

File: test/tests/services/query-builder.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      });
+  });
+
+  after((done) => {
+    mongoose.connection.close();
+    done();
+  });
+
+  describe('addJoinToQuery function', () => {
+    context('on correct inputs', () => {
```
Comment (SteveBunlon) [inline test/tests/services/query-builder.js]: What do you mean by inputs ?

File: test/tests/services/query-builder.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+      it('should format correctly', () => {
```
Comment (SteveBunlon) [inline test/tests/services/query-builder.js]: Do you mean it should format 'the join aggregate' ?

File: test/tests/services/query-builder.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+          },
+        };
+        const joins = [];
+        queryBuilder.addJoinToQuery(field, joins);
+        expect(joins).to.deep.include(expectedJoin);
+      });
+    });
+
+    context('on integration field', () => {
+      it('should not contains the join', () => {
```
Comment (SteveBunlon) [inline test/tests/services/query-builder.js]: 'Integration field should be ignored' seems more clear to me. 'should not contains the join' is a quite technical description related to your fix.

**Pull request merged**